<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_Welcome</class>
 <widget class="QDialog" name="Dlg_Welcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color:rgb(255,255,255);
}
QPushButton
{
background:rgba(0,0,0,0);

}
QPushButton#btnClose
{
image:url(:/image/delet_button.png);
color:white;
}
QPushButton#btnClose:hover
{
image:url(:/image/delet_button_pressed.png);
color:white;
}
QPushButton#btnSign
{
color:grey;
}
QPushButton#btnSign:hover
{
color:black;
}
QPushButton#btnLog
{
image:url(:/image/login.png);
color:white;
}
QPushButton#btnLog:hover
{
image:url(:/image/login-pressed.png);
color:white;
}
QRadioButton
{

}
QPushButton#pushButton
{
color:grey;
}
QPushButton#pushButton:hover
{
color:black;
}
QLineEdit
{
border:none;
border-bottom:1px solid rgb(229,229,229);
font:normal 12px;
color:rgb(154,154,154);
}
QLineEdit:hover
{
border:none;
border-bottom:1px solid rgb(0,0,0);
font:normal 12px;
color:rgb(154,154,154);
}
QLineEdit:focus
{
border:none;
border-bottom:1px solid rgb(0,0,0);
font:normal 12px;
color:rgb(0,0,0);
}
QLabel#label_2
{
background:rgba(0,0,0,0);
image:url(:/image/world_beyond_integrated_back.png);
}
QLabel#label_3
{

image:url(:/image/Logo.png);
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>430</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="LeAccount">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>60</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LePassword">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>95</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLog">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>160</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSign">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rdbtnUser">
       <property name="text">
        <string>用    户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdbtnAdmin">
       <property name="text">
        <string>管理员</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <hintingpreference>PreferDefaultHinting</hintingpreference>
        </font>
       </property>
       <property name="text">
        <string>忘记密码</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnSign_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>简约版</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>430</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Logo.qrc">:/image/LoginTop.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>20</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>100</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rdbtnUser</tabstop>
  <tabstop>rdbtnAdmin</tabstop>
  <tabstop>LeAccount</tabstop>
  <tabstop>LePassword</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>btnLog</tabstop>
  <tabstop>btnSign</tabstop>
 </tabstops>
 <resources>
  <include location="Logo.qrc"/>
 </resources>
 <connections/>
</ui>
